src/styles.scss

```diff
@@ -1,4 +1,6 @@
 @import "./cc-primary-theme.scss"; /* You can add global styles to this file, and also import other style files */
+@import '~css-star-rating/scss/star-rating';
```

Comment:
Do we need this?

Reply:
Yes - https://github.com/BioPhoton/angular-star-rating/wiki/Installing